mmv1/third_party/terraform/website/docs/d/oracle_database_db_nodes.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Oracle Database"
+description: |-
+  List all database nodes of a Cloud VmCluster.
+---
+
+# google_oracle_database_db_nodes
+
+List all DbNodes of a Cloud VmCluster.
+
+## Example Usage
+
+```hcl
+data "google_oracle_database_db_nodes" "my_db_nodes"{
+	location = "us-east4"
+	cloud_vm_cluster = "vmcluster-id"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `cloud_vm_cluster` - (Required) The ID of the VM Cluster.
+
+* `location` - (Required) The location of the resource.
+
+- - -
+* `project` - (Optional) The project in which the resource belongs. If it
+    is not provided, the provider project is used.
+
+* `db_nodes` - (Output) List of dbNodes.
```

Comment:
Missing "structure is documented below" link - see https://github.com/hashicorp/terraform-provider-google/blob/e65430eedacbbd7a89d31208ef578840b53054e7/website/docs/d/dns_keys.html.markdown?plain=1#L55 for example.

Note that the nested properties should come after all properties at a given level to avoid ambiguity about whether something is nested or not -  in this case it looks like the order is unambiguous anyway but it would be nice to have the links for consistency.